src/services/DappsStakingEvents.ts

```diff
@@ -541,15 +543,39 @@ export class DappsStakingEvents extends ServiceBase implements IDappsStakingEven
         }
     }
 
+    public async getAggregatedPeriodData(network: NetworkType, period: number): Promise<PeriodDataResponse[]> {
+        if (!['shibuya'].includes(network)) {
```

Comment:
perhaps this is better ?

if(network === 'shibuya' ){
       throw new Error(`This method is not supported for the network ${network}`);
}

Reply:
It is, but i will add other networks soon. That's why I put array :)